yjit/src/codegen.rs

```diff
@@ -2698,24 +2716,29 @@ fn gen_opt_gt(
 }
 
 // Implements specialized equality for either two fixnum or two strings
-// Returns true if code was generated, otherwise false
+// Returns None if enough type information isn't available, Some(true)
+// if code was generated, otherwise false.
```

Comment:
Minor nitpick, but...

"otherwise Some(false)"?

Reply:
thanks for catching it :+1: https://github.com/ruby/ruby/pull/7168/commits/25d7a714dece48417ba8a5ee7108e9e8af076a40